zkevm-circuits/src/evm_circuit/execution.rs

```diff
@@ -208,6 +208,7 @@ pub(crate) struct ExecutionConfig<F> {
     selfbalance_gadget: SelfbalanceGadget<F>,
     sha3_gadget: Sha3Gadget<F>,
     shl_shr_gadget: ShlShrGadget<F>,
+    staticcall_gadget: CallStaticCallGadget<F, false>,
```

Comment:
I would prefer also use `CallGadget` or `CallOpGadget` as future `CallCode` may embed in this gadget as well.

Reply:
Fixed to rename file `call_staticcall.rs` to `callop.rs`, and `CallStaticCallGadget` to `CallOpGadget`.